modules/claims_api/lib/bd/bd.rb

```diff
@@ -10,8 +10,7 @@ module ClaimsApi
   # Takes an optional request parameter
   # @param [] rails request object
   class BD
-    def initialize(request: nil)
-      @request = request
+    def initialize
```

Comment:
The if/else used to set the base_name variable in the client method still references "@request".